README.md

```diff
@@ -44,7 +44,8 @@ Markor is a TextEditor for Android. This project aims to make an editor that is
 <br/>🌎 Language selection -- use other language than on the system
 <br/>
 <br/>🔃 Markor is an offline app. It works with sync apps, but they have to do syncing respectively. Sync clients known to work in combination include BitTorrent Sync, Dropbox, FolderSync, OwnCloud, NextCloud, Seafile, Syncthing, Syncopoli
-<br/>🔒 Can encrypt your textfiles with AES256. You need to set a password at the settings and use Android device with version KitKat or newer. You can use [jpencconverter](https://gitlab.com/opensource21/jpencconverter) to encrypt/decrypt easily on desktop. Be aware that only the text is encrypted not pictures or attachments.  
+<br/>🔒 Can encrypt your textfiles with AES256. You need to set a password at the settings and use Android device with version KitKat or newer. You can use [jpencconverter](https://gitlab.com/opensource21/jpencconverter) to encrypt/decrypt easily on desktop. Be aware that only the text is encrypted not pictures or attachments.
+**Hint** This protects you content from other apps or person who are able to read your sd-card. It's **no** protection against anyone who was able to open the markor-app.  
```

Comment:
I don't think this Hint notice is necessary. If your device is under somebody's (unwanted) control, everything is under their control - nothing specific to Markor.

Reply:
I don't think that there only exists total control or no control. Every file-manager is normally able to read the sd-card and can send everything to a server. Against this it's an effective protection. There are other app like keepass, which protect the data even if someone open the app or is able to make screenshots. So I think it's a good idea to make this clear in the readme. Obviously for everyone how use the app should be clear, that if he gives someone the phone and the person open the markor app, that the other person can read the private notes. But therefore you must install the app.

Reply:
I understand, but password for for opening the app ("app lock") and encrypting files with a password are two complete different things